server/netty-server/src/test/scala/sttp/tapir/server/netty/NettyFutureServerTest.scala

```diff
@@ -19,7 +19,7 @@ class NettyFutureServerTest extends TestSuite with EitherValues {
           val interpreter = new NettyFutureTestServerInterpreter(eventLoopGroup)
           val createServerTest = new DefaultCreateServerTest(backend, interpreter)
 
-          val tests = new AllServerTests(createServerTest, interpreter, backend, staticContent = false, multipart = false).tests()
+          val tests = new AllServerTests(createServerTest, interpreter, backend, staticContent = true, multipart = false).tests()
```

Comment:
you can simply remove `staticContent = true` as it's the default

Reply:
updated